CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.0.12] - 2019-05-24
+* Fixed issue with Speck not responding to preset changes
```

Comment:
Put this under A `### Fixed` heading, please!

Reply:
(take a look at the rest of our CHANGELOG to see what it should look like)

Reply:
whoops fixed

Reply:
Also please make sure these have periods at the end, like the rest of the CHANGELOG entries. (Small detail, but it's good to be as consistent as possible!)